allennlp/tests/common/file_utils_test.py

```diff
@@ -4,10 +4,14 @@
 import pathlib
 import json
 
+from botocore.exceptions import ClientError
 import pytest
 import responses
 
-from allennlp.common.file_utils import url_to_filename, filename_to_url, get_from_cache, cached_path
+from allennlp.common.file_utils import url_to_filename, filename_to_url, get_from_cache, \
+    cached_path, \
+    split_s3_path, \
+    s3_request
```

Comment:
we tend to prefer using parenthesis rather than backslashes for line continuations

```
from allennlp.common.file_utils import (url_to_filename, ... 
)
```